diem-move/diem-framework/DPN/releases/artifacts/current/build/DiemCoreFramework/sources/Genesis.move

```diff
@@ -39,6 +39,7 @@ module DiemFramework::Genesis {
         chain_id: u8,
         initial_diem_version: u64,
         consensus_config: vector<u8>,
+        _config_storage_signer: signer,
```

Comment:
I'm wondering if this should just be the `dr_account`/if that should be renamed and this signer should be removed.

My thinking being that the core framework needs both a fixed `CoreFramework` address for the module, but also a fixed `CoreResources` address that contains all of the core resources for the network since the Rust code will expect specific access paths under a specific address.

Reply:
Yes agreed that we should have separate fixed `CoreFramework` and `CoreResources` addresses. For now I guess we can use dr's address as `CoreResources`, but in the long run dr's address (0xA550C18) should be replaced with something else right? Since association is a DPN specific concept.

Reply:
Agree. I think for now it makes it simpler to use the DR address, but once the refactors are done and everything is working changing the address value would make sense.